packages/dx-grid-core/src/plugins/local-selection/computeds.js

```diff
@@ -0,0 +1,16 @@
+export const getAvailableToSelect = (rows, getRowId, isGroupRow) => {
+  let dataRows = rows;
+  if (isGroupRow) {
+    dataRows = dataRows.filter(row => !isGroupRow(row));
+  }
+  return dataRows.map(row => getRowId(row));
+};
+
+export const isSomeSelected = ({ selection, availableToSelect }) =>
```

Comment:
The `is` prefix looks a bit weird here because `some` presumes that there can be multiple items.
Why do all computed names start with a verb? We have used nouns before.